StardewArchipelago/Locations/CodeInjections/Modded/SVE/SVECutsceneInjections.cs

```diff
@@ -157,5 +169,16 @@ public static void FixMonsterSlayerQuest_IncludeReleaseofGoals_Postfix()
                 return;
             }
         }
+
+        public static string AuroraVineyardMatchesBundles()
+        {
+            
+            var bundlesPrice = _archipelago.SlotData.BundlePrice;
+            if (bundlesPrice.HasFlag(BundlePrice.Normal))
```

Comment:
Why would you treat this array as flags?

Reply:
Will need better clarity on this.  You likely know more on what the structure is than me.  My assumption is that its the setting of the player.